docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,640 @@
+6. `upl` (OPTIONAL, string). If this algorithm is part of a product offered by the Dutch Government,
+    it should contain a URI from the [Uniform Product List](https://standaarden.overheid.nl/owms/oquery/UPL-actueel.plain).
+7. `owners` (list). There can be multiple owners. For each owner the following fields are present.
+
+    1. `organization` (REQUIRED, string). Name of the organization that owns the model.
+    2. `oin` (OPTIONAL, string). If applicable the [Organisatie-identificatienummer (OIN)](https://www.logius.nl/domeinen/toegang/organisatie-identificatienummer/wat-is-het).
+    3. `name` (OPTIONAL, string). Name of a contact person within the organisation.
+    4. `email` (OPTIONAL, string). Email address of the contact person or organization.
+    5. `role` (OPTIONAL, string). Role of the contact person. This field should only be set when the `name` field
+    is set.
+
+### 2. `model_index`
+
+There can be multiple models. For each model the following fields are present.
+
+1. `name` (REQUIRED, string). The name of the model.
+2. `model` (REQUIRED, string). A URI pointing to a repository containing the model file.
+3. `results` (list). There can be multiple results. For each result the following fields are present.
+
+    1. `task` (list).
+
+        1. `task_type` (REQUIRED, string). The task of the model, for example "object-classifcation".
+        2. `task_name` (OPTIONAL, string). A pretty name fo the model taks, for example "Object Classification".
+
+    2. `dataset` (list).
+
+        1. `type` (REQUIRED, string). The type of the dataset, can be a dataset id from [HuggingFace datasets](https://hf.co/datasets)
+        or a link to a repository containing the dataset[^1], for example "common_voice".
+        2. `name` (REQUIRED, string). Name pretty name for the dataset, for example "Common Voice (French)".
+        3. `split` (OPTIONAL, string). The split of the dataset, for example "train".
+        4. `revision` (OPTIONAL, string). Version of the dataset, for example 5503434ddd753f426f4b38109466949a1217c2bb.
+
+    3. `metrics` (list). There can be multiple metrics. For each metric the following fields are present.
+
+        1. `type` (REQUIRED, string). A metric-id from [Hugging Face metrics](https://hf.co//metrics)[^2], for example accuracy.
+        2. `name` (REQUIRED, string). A descriptive name of the metric. For example "false positive rate" is
+        not a descriptive name, but "training false positive rate w.r.t class x" is.
+        3. `value` (REQUIRED, float). The value of the metric.
+        4. `class` (OPTIONAL, string/int/float/bool). Some metrics (such as false positive rate for multiclass classification)
+        depend on a specific output class. In this field the output class can be specified. It must only be set for metrics
+        for which it makes sense.
+        5. `subgroups` (list). Metrics can be computed on subgroups of specific features. For example one can compute the
```

Comment:
Hmm, can you give an example of a metric that would only be relevant for certain model parameters?

Reply:
@ChristopherSpelt an example can be that you want to store metrics per epoch to report about development of a metric during the training of a model. This could be used as part of validating a model.